components/rendering/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Vincent Prouillet <prouillet.vincent@gmail.com>"]
 [dependencies]
 tera = { version = "1.0.0-beta.10", features = ["preserve_order"] }
 syntect = "=3.2.0"
-pulldown-cmark = "0.5"
+pulldown-cmark = "0.6"
```

Comment:
I think it might be used in some other sub crates as well? Can't remember

Reply:
You are right! I'll update those too.